### PR TITLE
Release notes for iOS SDK release 6.10.0

### PR DESCRIPTION
**Pull request by umairali332**

### CODE REVIEW

**Comment by umairali332** [comment]
@j9t @itay1313 Hey we are looking to release iOS SDK 6.10.0. Please review this and suggest any changes if necessary. Thanks

File: pages/Documents/MessagingChannels/MobileAppMessagingSDKforiOS/ReleaseNotes/Regular/6.10.0.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+- Expose a config for setting minimum height of the structured content buttons.
+- Expose a config for setting structure content button border's corner radius.
+- Expose a config for setting structured content button of link type foreground color.
+
+#### New configurations
+
+##### structuredButtonMinimumHeight
+- **Description:** Set the minimum height constraint for Structured Constant button elements
+- **Type:** CGFloat?
+- **Default Value:**  nil
```
Comment (j9t) [inline pages/Documents/MessagingChannels/MobileAppMessagingSDKforiOS/ReleaseNotes/Regular/6.10.0.md:45]: Please replace double by single spaces (within content of entire file—same issue for other default values).

File: pages/Documents/MessagingChannels/MobileAppMessagingSDKforiOS/ReleaseNotes/Regular/6.10.0.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+- **Available since:** 6.10.0
+
+##### structuredContentButtonBorderCornerRadius
+- **Description:** Set the corner radius of Structured Constant button elements border
+- **Type:** CGFloat
+- **Default Value:**  15.0
```
Comment (j9t) [inline pages/Documents/MessagingChannels/MobileAppMessagingSDKforiOS/ReleaseNotes/Regular/6.10.0.md:51]: Change to “Default value”—although there are many pages using title case, we do use sentence case; this also seems consistent with previous release notes.

(Change for all occurrences in this file.)

File: pages/Documents/MessagingChannels/MobileAppMessagingSDKforiOS/ReleaseNotes/Regular/6.10.0.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+- **Available since:** 6.10.0
+
+##### structuredContentLinkButtonTextColor
```
Comment (j9t) [inline pages/Documents/MessagingChannels/MobileAppMessagingSDKforiOS/ReleaseNotes/Regular/6.10.0.md:54]: Suggestion: This looks like code, and if it is, consider backticks to mark it as code. Like:

```
##### `structuredContentLinkButtonTextColor`
````

Comment (j9t) [reply pages/Documents/MessagingChannels/MobileAppMessagingSDKforiOS/ReleaseNotes/Regular/6.10.0.md:54]: (Affects other headings, too.)

**Review by j9t — CHANGES_REQUESTED** [review]
There would be some more stylistic issues, but I take it this is otherwise consistent with previous release notes?

File: pages/Documents/MessagingChannels/MobileAppMessagingSDKforiOS/ReleaseNotes/Regular/6.10.0.md (hunk 1 above)
Comment (umairali332) [reply pages/Documents/MessagingChannels/MobileAppMessagingSDKforiOS/ReleaseNotes/Regular/6.10.0.md]: Done.

File: pages/Documents/MessagingChannels/MobileAppMessagingSDKforiOS/ReleaseNotes/Regular/6.10.0.md (hunk 2 above)
Comment (umairali332) [reply pages/Documents/MessagingChannels/MobileAppMessagingSDKforiOS/ReleaseNotes/Regular/6.10.0.md]: Done.

File: pages/Documents/MessagingChannels/MobileAppMessagingSDKforiOS/ReleaseNotes/Regular/6.10.0.md (hunk 3 above)
Comment (umairali332) [reply pages/Documents/MessagingChannels/MobileAppMessagingSDKforiOS/ReleaseNotes/Regular/6.10.0.md]: Done.

**Comment by umairali332** [comment]
@j9t We did the release and we are good to merge this now. Thanks

**Pull request merged**